dnd-bot/dnd_bot/dc/ui/views/view_game.py

```diff
@@ -348,8 +348,75 @@ async def character_view_skills(self, button: nextcord.ui.Button, interaction: n
 
 
+class ViewManageItems(ViewGame):
+    """ view for managing player's equipment"""
+    def __init__(self, token, user_discord_id):
+        super().__init__(token, user_discord_id)
+
+        self.player = Multiverse.get_game(token).get_player_by_id_user(user_discord_id)
+
+        if len(self.player.backpack) > 0:
+            item_select_options = []
+
+            for count, item in enumerate(self.player.backpack):
+                item_select_options.append(nextcord.SelectOption(
+                    label=f"{item.name}",
+                    value=str(count)  # TODO should be item.id, but for now database is not functioning properly
+                ))
+            self.select_list = nextcord.ui.Select(
+                placeholder="Choose an item to manage",
+                options=item_select_options,
+                row=0
+            )
+
+            self.add_item(self.select_list)
+
+            equip_button = Button(label='Equip item', style=nextcord.ButtonStyle.blurple, custom_id='manage-items-equip'
+                                                                                                    '-button')
+            equip_button.callback = self.equip
+            self.add_item(equip_button)
+
+            remove_button = Button(label='Remove item', style=nextcord.ButtonStyle.blurple,
+                                   custom_id='manage-items-remove-button')
+            remove_button.callback = self.remove
+            self.add_item(remove_button)
+
+        cancel_button = Button(label='Cancel', style=nextcord.ButtonStyle.red, row=1, custom_id='attack-cancel-button')
+        cancel_button.callback = self.cancel
+        self.add_item(cancel_button)
+
+    async def equip(self, interaction: nextcord.Interaction):
+        await HandlerManageItems.equip_item(self.player, int(self.select_list.values[0]))
+        embed = MessageTemplates.equipment_message_template(self.player)
+        await Messager.edit_last_user_message(user_id=interaction.user.id, embeds=[embed],
+                                              view=ViewManageItems(self.token, interaction.user.id))
+
+    async def remove(self, interaction: nextcord.Interaction):
+        await HandlerManageItems.remove_item(self.player, int(self.select_list.values[0]))
+        embed = MessageTemplates.equipment_message_template(self.player)
+        await Messager.edit_last_user_message(user_id=interaction.user.id, embeds=[embed],
+                                              view=ViewManageItems(self.token, interaction.user.id))
+
+    async def cancel(self, interaction: nextcord.Interaction):
+        turn_view_embed = MessageTemplates.equipment_message_template(self.player)
+
+        self.game.players_views[self.user_discord_id] = (ViewMain, [])
+        await Messager.edit_last_user_message(user_id=interaction.user.id, embeds=[turn_view_embed],
+                                              view=ViewEquipment(self.token, interaction.user.id))
+
+
```

Comment:
wg mnie w tym menu powinno być tak:
- lista itemów do selectowania powinna zawierać zarówno plecak jak i ekwipunek
- itemy wyekwipowane w jakiś sposób oznaczone
- przyciski pojawiają się / są dostępne dopiero po wybraniu przedmiotu
  - itemu wyekwipowanego nie można usunąć, można tylko wyekwipować (a potem może zobaczyć statystyki i opis)
  - item z plecaka tak jak teraz

Reply:
bo to dziwne że można tylko zamieniać itemy, a nie unequpowac

Reply:
przenoszę na issue